requirements-workflow.txt

```diff
@@ -1,4 +1,6 @@
 sqlmodel
 PyYaml
 paramiko
-mlflow
\ No newline at end of file
+mlflow
+pg8000
+simple-slurm
```

Comment:
this was removed by https://github.com/AllenInstitute/ophys_etl_pipelines/pull/567